main.go

```diff
@@ -26,8 +28,19 @@ const (
 func mainInternal(w io.Writer) error {
 	fs := osfs.New("")
 
-	const logfilePerm = 0755
-	logfile, err := os.OpenFile(pocketbook.Share(logFileName), os.O_CREATE|os.O_APPEND, logfilePerm)
+	paths := [3]string{
```

Comment:
Good point @iMega, but too many operations to check and create folders:
- Better use go-billy fs abstraction.
- `pocketbook.ConfigPath` and `pocketbook.Share` has no required params.
- There is no need to get the folder stat to check that it doesn't exist.  It's better to try to create a folder and check an error when it exists.

```golang
if err := fs.MkdirAll(pocketbook.ConfigPath(), perm); err != nil && !os.IsExist(err) {
	return fmt.Errorf("failed to configuration folder: %w", err)
}

if err := fs.MkdirAll(pocketbook.Share(), perm); err != nil && !os.IsExist(err) {
	return fmt.Errorf("failed to shared folder: %w", err)
}
```

Reply:
I removed os.Stat as it is used internally by os.MkdirAll.

Reply:
@iMega Pay attention the error check should look like this:
```golang
err != nil && !os.IsExist(err)
```

Reply:
Could you explain me I can't understand why, please.

Reply:
Oh, sorry. My mistake. I've confused behaviour in go with other languages. 
Your variant is right.

Thanks.